### ISSUE TITLE
Allow to preview markdown content before sending comments and messages

### ISSUE THREAD

**Issue by willemarcel** [hotosm/tasking-manager#4069]
We have some comment and message forms that supports markdown content. It would be good to allow the users to preview the content before posting, in order to avoid markdown formatting errors.

**Comment by wireguy** [comment]
associated markdown enhancements:
- have some helper text to show what markdown is available to use
osm shows this, but you may not want to use the space (maybe a popup with text?)
![image](https://user-images.githubusercontent.com/47227325/104020978-0240bb00-518c-11eb-95e3-f5acae8e4a40.png)

**Comment by HelNershingThapa** [comment]
> @HelNershingThapa is this issue resolved by #5094?

Nope, only the project detail comment section has the preview thing as of now.
